src/App.tsx

```diff
@@ -38,8 +66,15 @@ function App() {
       {keycloak.authenticated ? (
         <>
           <Route path="/home" element={<Home />} />
-          <Route path="/caretaker/form" element={<CaretakerForm onSubmit={() => {}} />} />
+          <Route
+            path="/caretaker/form"
+            element={<CaretakerForm onSubmit={() => {}} />} //TODO: dodaj onSubmit
```

Comment:
Nie będzie trzeba dodawać `onSubmit`, bo w zależności od tego co będzie w local storage to będzie wiadomo czy ma się tworzyć nowy profil czy edytować istniejący, także to zniknie z propsów, już z resztą mam to na branchu swoim.
```suggestion
            element={<CaretakerForm />}
```